### PR TITLE
LPS-165333 Move FileEntryContentDashboardItemTest unit test to integration test

### PR DESCRIPTION
**Pull request by Mikellorza**
Motivation
=======
Move JournalArticleContentDashboardItemTest unit test to integration test
[Link to story or ticket](https://issues.liferay.com/browse/LPS-165333)

Proposed Solution
========
Delete unit test and add integration test

Steps to Verify:
----------------
1-Execute integration test

### CODE REVIEW

**Comment by Mikellorza** [comment]
ci:test:sf

**Comment by Mikellorza** [comment]
ci:test:relevant

File: modules/apps/content-dashboard/content-dashboard-document-library-test/src/testIntegration/java/com/liferay/content/dashboard/document/library/internal/item/item/test/FileEntryContentDashboardItemTest.java [hunk 1]
```diff
@@ -0,0 +1,702 @@
...
+					_contentDashboardItemFactory.create(
+						fileEntry.getFileEntryId());
+
+		Assert.assertFalse(
+			versionableContentDashboardItem.isViewable(
+				_getMockHttpServletRequest()));
+	}
+
+	@Test
+	public void testViewVersionURL() throws Exception {
```
Comment (cgoncas) [inline modules/apps/content-dashboard/content-dashboard-document-library-test/src/testIntegration/java/com/liferay/content/dashboard/document/library/internal/item/item/test/FileEntryContentDashboardItemTest.java:551]: Following Liferay naming convention, the name of the test should be getViewVersionsURL.

**Review by cgoncas — CHANGES_REQUESTED** [review]
LGTM, only a small rename is needed, I think.

**Review by jkappler — COMMENTED** [review]
same comment as @cgoncas

File: modules/apps/content-dashboard/content-dashboard-document-library-test/src/testIntegration/java/com/liferay/content/dashboard/document/library/internal/item/item/test/FileEntryContentDashboardItemTest.java (hunk 1 above)
Comment (Mikellorza) [reply modules/apps/content-dashboard/content-dashboard-document-library-test/src/testIntegration/java/com/liferay/content/dashboard/document/library/internal/item/item/test/FileEntryContentDashboardItemTest.java]: done, thanks for your feedback @cgoncas :)

**Comment by Mikellorza** [comment]
Perfect, thanks @jkappler  :)

**Comment by jkappler** [comment]
ci:forward

**Comment by jkappler** [comment]
ci:stop

**Comment by Mikellorza** [comment]
ci:test:sf

**Comment by Mikellorza** [comment]
ci:test:relevant

**Comment by jkappler** [comment]
ci:forward

**Pull request closed without merging**